staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer.go

```diff
@@ -219,10 +219,20 @@ func (az *Cloud) EnsureLoadBalancerDeleted(ctx context.Context, clusterName stri
 
 // GetLoadBalancerName returns the LoadBalancer name.
 func (az *Cloud) GetLoadBalancerName(ctx context.Context, clusterName string, service *v1.Service) string {
-	// TODO: replace DefaultLoadBalancerName to generate more meaningful loadbalancer names.
+	if az.LoadBalancerName != "" {
+		return az.LoadBalancerName
+	}
```

Comment:
Please rollback the changes in L222-L224. The name here is used for frontendipconfigurations, service UID should be used so that different services configurations won't be conflicted.

Reply:
Thanks for reviewing. The corresponding codes have been fixed.